src/controllers/pods.ts

```diff
@@ -153,6 +153,11 @@ export default function (server: Hapi.Server, deps: Injector) {
         `manifestHash=${manifestHash}`)
     }
 
+    if (manifest.privateManifest) {
+      throw Boom.serverUnavailable('pod has stopped. ' +
```

Comment:
I think this can have the actual error since it will not be send in the response but just in the hosts logs. Are we ok with logging that the pod is private in the hosts logs? That seems ok to me.

Reply:
That should be fine